projects/OdroidXU3/packages/Mali_OpenGL_ES_SDK/package.mk

```diff
@@ -41,5 +41,9 @@ make_target() {
 
 makeinstall_target() {
   mkdir -p $SYSROOT_PREFIX/usr/include
-    cp -PR usr/include/* $SYSROOT_PREFIX/usr/include
+    cp -PR $PKG_BUILD/inc/* $SYSROOT_PREFIX/usr/include
+    cp -PR $PKG_BUILD/simple_framework/inc/mali/* $SYSROOT_PREFIX/usr/include
+
+    mkdir -p $SYSROOT_PREFIX/usr/lib/pkgconfig
+    cp -PRv $PKG_DIR/pkgconfig/*.pc $SYSROOT_PREFIX/usr/lib/pkgconfig
```

Comment:
All these pkgconfig files are not needed, I was able to build and link RA without them